<ui version="4.0" >
 <class>QRL_TargetsManager</class>
 <widget class="QDialog" name="QRL_TargetsManager" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>303</width>
    <height>228</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Targets Manager</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qrtailab.qrc" >
    <normaloff>:/icons/connect_icon.xpm</normaloff>:/icons/connect_icon.xpm</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidgetet" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Start Target</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" colspan="2" >
        <widget class="QListWidget" name="listWidget" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QPushButton" name="pushButton_4" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>>></string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item rowspan="2" row="0" column="3" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="title" >
          <string>Start local Target</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Target</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2" >
           <widget class="QLineEdit" name="lineEdit" />
          </item>
          <item row="2" column="0" colspan="2" >
           <widget class="QCheckBox" name="checkBox_3" >
            <property name="text" >
             <string>auto. connect</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QCheckBox" name="checkBox" >
            <property name="text" >
             <string>wait</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" >
           <widget class="QCheckBox" name="checkBox_2" >
            <property name="text" >
             <string>stop after</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QLineEdit" name="lineEdit_2" />
          </item>
          <item row="5" column="0" colspan="2" >
           <widget class="QPushButton" name="pushButton_2" >
            <property name="text" >
             <string>Start Target</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="pushButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Add Target</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="pushButton_3" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Remove</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Status</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>Target</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="targetNameLineEdit" >
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="label_11" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Output:</string>
            </property>
           </widget>
          </item>
          <item rowspan="2" row="0" column="3" >
           <widget class="QTextEdit" name="textEdit" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_12" >
            <property name="text" >
             <string>Status:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="targetStatusLineEdit" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>not running</string>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Connect to Target</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7" >
       <item row="0" column="1" >
        <widget class="QGroupBox" name="targetGroupBox" >
         <property name="title" >
          <string>Target</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5" >
          <item row="0" column="0" >
           <widget class="QLabel" name="targetNameLabel" >
            <property name="text" >
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="targetNameLineEdit_2" >
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2" >
           <widget class="QGroupBox" name="statusGroupBox" >
            <property name="title" >
             <string>Status</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_8" >
             <item row="1" column="0" >
              <widget class="QLabel" name="targetConnectedLabel" >
               <property name="text" >
                <string>not connected</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" >
              <widget class="QLabel" name="targetRunningLabel" >
               <property name="text" >
                <string>not running</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QGroupBox" name="hartRTGroupBox" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="toolTip" >
             <string>Can only changed at startup.</string>
            </property>
            <property name="title" >
             <string>hard real time mode</string>
            </property>
            <property name="checkable" >
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout" >
             <item row="0" column="0" >
              <widget class="QCheckBox" name="scopesHRTCheckBox" >
               <property name="toolTip" >
                <string>Let the scope threads run in hard real time mode. 
Can block the cpu, if to many data are transmitted.

 Important for the saving process. Prevent saving from blocking and data loss.</string>
               </property>
               <property name="text" >
                <string>Scopes</string>
               </property>
               <property name="checked" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <widget class="QCheckBox" name="logsHRTCheckBox" >
               <property name="toolTip" >
                <string>Let the log threads run in hard real time mode. 
Can block the cpu, if to many data are transmitted.

 Important for the saving process. Prevent saving from blocking and data loss.</string>
               </property>
               <property name="text" >
                <string>Logs</string>
               </property>
               <property name="checked" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0" >
              <widget class="QCheckBox" name="aLogsHRTCheckBox" >
               <property name="toolTip" >
                <string>Let the scope threads run in hard real time mode. 
Can block the cpu, if to many data are transmitted.

 Important for the saving process. Prevent saving from blocking and data loss.</string>
               </property>
               <property name="text" >
                <string>ALogs</string>
               </property>
               <property name="checked" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="4" column="0" >
           <spacer name="verticalSpacer" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
         <zorder>targetNameLabel</zorder>
         <zorder>targetNameLineEdit_2</zorder>
         <zorder>statusGroupBox</zorder>
         <zorder>hartRTGroupBox</zorder>
         <zorder>verticalSpacer</zorder>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QGroupBox" name="connectGroupBox" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="title" >
          <string>Connect to Target</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6" >
          <item rowspan="5" row="0" column="0" >
           <widget class="QGroupBox" name="targetParameterGroupBox" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="title" >
             <string>Parameter</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_9" >
             <item row="0" column="0" >
              <layout class="QVBoxLayout" >
               <property name="spacing" >
                <number>6</number>
               </property>
               <property name="margin" >
                <number>0</number>
               </property>
               <item>
                <layout class="QHBoxLayout" >
                 <property name="spacing" >
                  <number>6</number>
                 </property>
                 <property name="margin" >
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="ipLineEdit" >
                   <property name="text" >
                    <string>127.0.0.1</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="ipLabel" >
                   <property name="text" >
                    <string>IP Adress</string>
                   </property>
                   <property name="buddy" >
                    <cstring>ipLineEdit</cstring>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" >
                 <property name="spacing" >
                  <number>6</number>
                 </property>
                 <property name="margin" >
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="taskLineEdit" >
                   <property name="text" >
                    <string>IFTASK</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="taskLabel" >
                   <property name="text" >
                    <string>Task Identifier</string>
                   </property>
                   <property name="buddy" >
                    <cstring>taskLineEdit</cstring>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" >
                 <property name="spacing" >
                  <number>6</number>
                 </property>
                 <property name="margin" >
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="scopeLineEdit" >
                   <property name="text" >
                    <string>RTS</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="scopeLabel" >
                   <property name="text" >
                    <string>Scope Identifier</string>
                   </property>
                   <property name="buddy" >
                    <cstring>scopeLineEdit</cstring>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" >
                 <property name="spacing" >
                  <number>6</number>
                 </property>
                 <property name="margin" >
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="logLineEdit" >
                   <property name="text" >
                    <string>RTL</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="logLabel" >
                   <property name="text" >
                    <string>Log Identifier</string>
                   </property>
                   <property name="buddy" >
                    <cstring>logLineEdit</cstring>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" >
                 <property name="spacing" >
                  <number>6</number>
                 </property>
                 <property name="margin" >
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="alogLineEdit" >
                   <property name="text" >
                    <string>RAL</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="ALogLabel" >
                   <property name="text" >
                    <string>Alog Identifier</string>
                   </property>
                   <property name="buddy" >
                    <cstring>alogLineEdit</cstring>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" >
                 <property name="spacing" >
                  <number>6</number>
                 </property>
                 <property name="margin" >
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="ledLineEdit" >
                   <property name="text" >
                    <string>RTE</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="ledLabel" >
                   <property name="text" >
                    <string>Led Identifier</string>
                   </property>
                   <property name="buddy" >
                    <cstring>ledLineEdit</cstring>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" >
                 <property name="spacing" >
                  <number>6</number>
                 </property>
                 <property name="margin" >
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="meterLineEdit" >
                   <property name="text" >
                    <string>RTM</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="meterLabel" >
                   <property name="text" >
                    <string>Meter Identifier</string>
                   </property>
                   <property name="buddy" >
                    <cstring>meterLineEdit</cstring>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" >
                 <property name="spacing" >
                  <number>6</number>
                 </property>
                 <property name="margin" >
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="synchLineEdit" >
                   <property name="text" >
                    <string>RTY</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="synchLabel" >
                   <property name="text" >
                    <string>Synch Identifier</string>
                   </property>
                   <property name="buddy" >
                    <cstring>synchLineEdit</cstring>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QPushButton" name="connectPushButton" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="toolTip" >
             <string>Connect to a target with the given task identifier at the given IP adress.</string>
            </property>
            <property name="text" >
             <string>Connect</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
            <property name="default" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QPushButton" name="startPushButton" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="toolTip" >
             <string>If the target is waiting, then it can be started be clicking the button.</string>
            </property>
            <property name="text" >
             <string>start</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QPushButton" name="stopPushButton" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="toolTip" >
             <string>Stops the target and disconnect.</string>
            </property>
            <property name="text" >
             <string>stop</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QPushButton" name="disconnectPushButton" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="toolTip" >
             <string>Disconnect from the target, but the target is still running.</string>
            </property>
            <property name="text" >
             <string>Disconnect</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite" >
      <attribute name="title" >
       <string>Target Elements</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" >
       <item row="0" column="0" >
        <widget class="QTableWidget" name="targetTableWidget" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>16</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="text" >
        <string>Help</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>16</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>close</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qrtailab.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>QRL_TargetsManager</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>600</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel" >
     <x>381</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
